pkg/etcd/etcdprocess.go

```diff
@@ -28,6 +29,10 @@ var isTest = false
 func init() {
 	// For bazel
 	// TODO: Use a flag?
+
+	// used to fix glog parse error.
+	flag.CommandLine.Parse([]string{})
```

Comment:
We should move to klog, but ... thanks for fixing :-)

Reply:
Sure @justinsb . I'll keep a note to change to klog once I update the KOPS side of the DO changes.